asciidoc/client-applications.adoc

```diff
@@ -182,15 +148,49 @@ include::{javascript-examples}/examples.test.js[tags=config-custom-resolver]
 ----
 ======
 
-[.include-with-python]
-======
-[source, python]
-----
-include::{python-examples}/custom_resolver_example.py[tags=custom-resolver]
-----
-======
+// [.include-with-python]
+// ======
+// [source, python]
+// ----
+// include::{python-examples}/custom_resolver_example.py[tags=custom-resolver]
+// ----
+// ======
+====
+
+=== Routing table
+
+The routing table is the glue between the driver connectivity layer and the database surface.
```

Comment:
Is it the glue or "can it be viewed as" the glue?

Reply:
"acts like the glue"?